lms/djangoapps/onboarding/forms.py

```diff
@@ -371,11 +372,11 @@ def save(self, request, commit=True):
         if not is_currently_unemployed and organization_name:
             organization_to_assign = Organization.objects.filter(label__iexact=organization_name).first()
             if not organization_to_assign:
-                organization_to_assign = Organization.objects.create(label=organization_name)
+                organization_to_assign, organization_created = Organization.objects.create(label=organization_name), True
 
             prev_org = user_info_survey.organization
             user_info_survey.organization = organization_to_assign
-            user_info_survey.is_first_learner = organization_to_assign.users_count() == 0
+            user_info_survey.is_first_learner = (organization_to_assign.users_count() == 0 and organization_created)
```

Comment:
Either we should simplify this condition, since there are three thing we are checking

a) Integer count
b) boolean value i.e. False
c) `organization_created`

Or we can add comment about our fix